app/crypto/create-stx-tx.ts

```diff
@@ -0,0 +1,24 @@
+import BN from 'bn.js';
+import BigNumber from 'bignumber.js';
+import { deriveRootKeychainFromMnemonic } from '@blockstack/keychain';
+import { makeSTXTokenTransfer } from '@blockstack/stacks-transactions';
+
+import { stacksNetwork } from '../environment';
+import { deriveStxAddressKeychain } from './derive-address-keychain';
+
+interface CreateStxTxArgs {
+  mnemonic: string;
+  recipient: string;
+  amount: BigNumber;
+}
+
+export async function createStxTransaction({ mnemonic, recipient, amount }: CreateStxTxArgs) {
+  const rootNode = await deriveRootKeychainFromMnemonic(mnemonic);
+  const { privateKey } = deriveStxAddressKeychain(rootNode);
+  return await makeSTXTokenTransfer({
```

Comment:
`return await` is not needed here. any promise returned is by default awaited in an async function